find-closest-facility-from-point/README.md

```diff
@@ -0,0 +1,47 @@
+# Find closest facility from point
+
+Find a route to the closest facility from a location.
+
+![Image of Find Closest Facility from Point](find_closest_facility_from_point.png)
+
+## Use case
+
+Quickly and accurately determining the most efficient route between a location and a facility is a frequently encountered task. For example, a paramedic may need to know which hospital in the vicinity offers the possibility of getting an ambulance patient critical medical care in the shortest amount of time. Solving for the closest hospital to the ambulance's location using an impedance of "travel time" would provide this information.
+
+## How to use the sample
+
+Tap near any of the hospitals and a route will be displayed from that tapped location to the nearest hospital.
+
+## How it works
+
+1. Create a `ClosestFacilityTask` using a Url from an online network analysis service.
```

Comment:
```suggestion
1. Create a `ClosestFacilityTask` using an URL from an online network analysis service.
```

Reply:
generally, an acronym in doc is all caps. Most languages use all caps too, but not Kotlin. Kotlin [coding conventions for naming](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names) are pretty loose, but do call out `camel case` for types and variables that are named after acronyms (longer than two chars...)